dev/com.ibm.ws.sipcontainer/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -90,8 +90,10 @@
 		<AD id="sipTasksDispatcher" name="%sip.message.dispatcher" description="%sip.message.dispatcher.desc"
    	        ibm:reference="com.ibm.ws.sip.container.was.ExecutorMessageDispatchingHandler" ibm:type="pid"
             type="String" required="false"/>
-            
-	
+
+		<AD name="internal" description="internal" id="useNettyTransport"
+			required="false" type="Boolean" default="true" ibm:beta="true"/>
+
```

Comment:
Didn't we discuss that the ibm:beta="true" isn't necessary here since we are making this internal? Internal means it will never b e documented. ibm:beta=true means it will only be documented in the beta, but it won't actually be in this case because it is internal.

Reply:
ibm:beta will be removed